### PR TITLE
unit tests for svg utility components

### PR DESCRIPTION
**Pull request by christianvogt**
Adds some unit tests to the svg utility components.

Could not properly test components relying on newer react context api. From investigation it looks like i need to upgrade `enzyme` and / or the `enzyme-adapter-react-16`. Will look into this further once we  update with latest from console master or move to console repo.

### CODE REVIEW

**Review by karthikjeeyar — APPROVED** [review]
LGTM

File: frontend/public/extend/devconsole/shared/components/svg/__tests__/SvgDefsProvider.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ).toBe(state.defs.first.node);
+    expect(
+      defsWrapper
+        .childAt(1)
+        .first()
+        .props().children,
+    ).toBe(state.defs.second.node);
+  });
+
+  it('should render defs');
```
Comment (divyanshiGupta) [inline frontend/public/extend/devconsole/shared/components/svg/__tests__/SvgDefsProvider.spec.tsx]: I am not sure why do we have this here.

**Review by divyanshiGupta — CHANGES_REQUESTED** [review]
@christianvogt tests looks good. But there are a few lint errors. Can you fix them please?

File: frontend/public/extend/devconsole/shared/components/svg/__tests__/SvgDefsProvider.spec.tsx (hunk 1 above)
Comment (christianvogt) [reply frontend/public/extend/devconsole/shared/components/svg/__tests__/SvgDefsProvider.spec.tsx]: Forgot to remove. Had plans to do more tests but the issues with context testing with the current library versions prevents it.

Comment (divyanshiGupta) [reply frontend/public/extend/devconsole/shared/components/svg/__tests__/SvgDefsProvider.spec.tsx]: Okay, then remove this line and we can add the remaining tests later once the library is updated.

Comment (christianvogt) [reply frontend/public/extend/devconsole/shared/components/svg/__tests__/SvgDefsProvider.spec.tsx]: Done.

**Pull request merged**